docs/static-types.md

```diff
@@ -19,6 +19,8 @@ import me.shadaj.scalapy.py
 }
 ```
 
+**Note:** You should define a static type as a subclass, instead of a trait as shown in this example, if you are using Scala 3.
```

Comment:
Rewording suggestion: start with "If you are using Scala 3, ..." so that it's clear what this note is for from the beginning.